java/src/jmri/jmrix/loconet/LnMeterInitTask.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Nothing that needs to be done now, but this header is not relevant.

Reply:
I will resolve this now, rather than let it sit for later resolution.

Reply:
resolved in a recent commit.